src/components/History/Widget/WidgetCard.tsx

```diff
@@ -174,7 +174,8 @@ const WidgetCard = ({
   ): QueueTransactions | undefined => {
     if (
       transaction.actions[0].FunctionCall.method_name === "ft_transfer_call" ||
-      transaction.actions[0].FunctionCall.method_name === "rollback_intent"
+      transaction.actions[0].FunctionCall.method_name === "rollback_intent" ||
+      transaction.actions[0].FunctionCall.method_name === "native_on_transfer"
```

Comment:
Seems like we can move `transaction.actions[0].FunctionCall.method_name` to a variable